pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/DescriptorLookupCache.java

```diff
@@ -46,7 +46,7 @@
     private transient Map<String, DescribableModel<? extends Describable>> describableModelMap;
 
     public static DescriptorLookupCache getPublicCache() {
-        return Jenkins.getInstance().getInjector().getInstance(DescriptorLookupCache.class);
+        return Jenkins.getInstance().getExtensionList(DescriptorLookupCache.class).get(0);
```

Comment:
My suggestion was to use `ExtensionList.lookup(DescriptorLookupCache.class).get(...)` instead.